src/main/java/org/miracum/etl/fhirtoomop/FhirServerItemReader.java

```diff
@@ -91,6 +98,9 @@ private IQuery<Bundle> searchQuery() {
             .sort(new SortSpec(IAnyResource.SP_RES_ID))
             .offset(0)
             .count(getPageSize());
+    if (goldenMerging && resourceTypeName.equals(ResourceType.PATIENT.getDisplay())) {
```

Comment:
There should be a setting like `goldenMerging` but is a comma separated list of the "golden resource types"

Reply:
There'll just be `Location`, right? We can add that when location mapping is added.

Reply:
We are currently indexing all of these:
```
      - Patient
      - Location
      - Organization
      - Practitioner
      - PractitionerRole
```

Reply:
Are we going to need to map Practitioners and Organizations? They currently aren't used, but I guess could be mapped to `Provider` in OMOP.

Reply:
I thought we may as well have an `application.yaml` for this and have it be more dynamic to match the mdm resources